### ISSUE TITLE
Unable to connect device

### ISSUE THREAD

**Issue by CorentinHeroux** [anmoljain10/react-native-zebra-bluetooth-printer#21]
This code always returns "Unable to connect device" no matter what device is used (Android 5, 8, 10).
The address sent is the correct one (returned by pairedDevices()).
Zebra ZD421 printer

```
let deviceAddress = '00:07:4D:BD:27:2E';
RNZebraBluetoothPrinter.connectDevice(deviceAddress)
    .then((res) => { 
        console.log(res);
    })
    .catch((error) => console.log(error.message));
```

Has anyone experienced this problem or seen how to solve it ? Thanks

**Comment by crventra941** [comment]
I've the same issue with the same printer, Do you have any solution yet?

**Comment by Stroi** [comment]
I have the same problem with _ZD421_ , I have the correct MAC address, and when I try to connect from android to the printer it always returns `Error: Unable to connect device` , printing also doesn't work. Any solutions @crventra941  @CorentinHeroux ?

**Comment by giorgioff** [comment]
hi! same problem here. I see "unpaired" as device type value. but also after a connectDevice call, the issue isn't resolved

**Comment by ombogdan** [comment]
same problem

**Comment by emirhanselamet** [comment]
same problem. anyone fix the issue?

**Comment by ombogdanJoinToIt** [comment]
> same problem. anyone fix the issue?

yes. you need to make changes in android app. please write me in telegram ombogdan_bbx and I send you new archive
